accname/name/comp_labelledby.html

```diff
@@ -60,54 +59,8 @@ <h2>Tests for precedence: <code>aria-labelledby</code> vs. <code>aria-labeledby<
-<button aria-labeledby="span1" aria-label="foo" data-expectedlabel="label" data-testname="button's hidden referenced name (display:none) with aria-labeledby (non-standard spelling) supercedes aria-label" class="ex">
-  <span id="span1" style="display:none;">
-    <span id="span2" style="display:none;">label</span>
-  </span>
-x
-</button>
-
-<button aria-labeledby="span3" aria-label="foo" data-expectedlabel="label" data-testname="button's hidden referenced name (visibility:hidden) with aria-labeledby (non-standard spelling) supercedes aria-label" class="ex">
-  <span id="span3" style="visibility:hidden;">
-    <span id="span4" style="visibility:hidden;">label</span>
-  </span>
-  x
-</button>
-
-<button aria-labeledby="span4" aria-label="foo" data-expectedlabel="foo" data-testname="button's hidden referenced name (visibility:hidden) with hidden aria-labeledby traversal falls back to aria-label" class="ex">
-  <span id="span4">
-    <span id="span5" style="visibility:hidden;">label</span>
-  </span>
-  x
-</button>
-
-<!-- Step 2B: AriaLabeledby supercedes 2D: AriaLabel -->
-<a href="#" aria-labeledby="span6" aria-label="foo" data-expectedlabel="label" data-testname="link's aria-labeledby (non-standard spelling) name supercedes aria-label" class="ex">x</a>
-<span id="span6">label</span>
-
-<!-- Step 2C: Embedded Control labelling via aria-labeledby not defined in accname spec -->
-
-<!-- Step 2E: Host Language Label is superceded by 2B: AriaLabeledby -->
-<img alt="alt" aria-labeledby="n2" data-expectedlabel="second label" data-testname="img's aria-labeledby (non-standard spelling) supercedes alt attribute" class="ex" />
-
-<svg aria-labeledby="n2" data-expectedlabel="second label" data-testname="svg's aria-labeledby (non-standard spelling) supercedes title tag" class="ex">
-  <circle cx="5" cy="5" r="4">
-    <title>circle</title>
-  </circle>
-</svg>
-
-<label for="input1">label</label>
-<input type="text" id="input1" aria-labeledby="n2" data-expectedlabel="second label" data-testname="input with label for association is superceded by aria-labeledby (non-standard spelling)" class="ex" />
-
-<!-- Step 2F: Name From Content is superceded by 2B: AriaLabeledby -->
-<button aria-labeledby="n2" data-expectedlabel="second label" data-testname="button name from contents is superceded by aria-labeledby (non-standard spelling)" class="ex">x</button>
-
-<!-- Step 2G: Text Node is superceded by 2B: AriaLabeledby, also see wpt/accname/name/comp_text_node.html -->
-<h1 aria-labeledby="n2" data-expectedlabel="second label" data-testname="h1 text is superceded by aria-labeledby (non-standard spelling)" class="ex">x</h1>
-
 <!-- Step 2H: Recursive Name from Content, see wpt/accname/name/comp_name_from_content.html  -->
-<h3 data-expectedlabel="image link2 link3" data-testname="heading name from content for each child including two nested links using aria-labeledby (non-standard spelling) with nested image" class="ex">
+<h3>
```

Comment:
FYI I probably should have just deleted this one, but I think it's okay to r+ the single commit anyway given that he end goal of this PR is to be reverted.